### ISSUE TITLE
Increasing Data

### ISSUE THREAD

**Issue by dnmacdon** [brainhack-school2020/ReadingTheMindInTheEyes---computer-vision#1]
As a followup to my comments in the presentation doc, just wanted to suggest you look into transformations (such as flipping images) that may increase the amount of data, since you mentioned that you're concerned about the size of your data set.

**Comment by olivleone** [comment]
Yes, thank you! This is a good idea and something I will play around with. 
A general question- say I'm still not able to produce a large enough data set, but I'm still interested in familiarizing with the tools and seeing what I can do and exactly how this fact will affect my outcome... Would you say it's not worth while even trying to learn with a small data set, or is there still a lot I can do even if the outcome isn't "successful"? (It seems like you might have experience in this type of work?)- thanks!

**Comment by ValiquetteVanessa** [comment]
@dnmacdon Playing with the size as well or even how blurry is the image could maybe work? But that would mean that @olivleone you would have to do some photoshopping? Or maybe there is a control dataset that you could merge? Like an object dataset or animals?
